Sources/tinys3/responses/S3ListMultipartUploadResponse.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+#if canImport(FoundationXML)
+import FoundationXML
+#endif
+
+struct S3ListMultipartUploadResponse {
+
+    struct S3MultipartUpload {
+        let key: String
+        let uploadId: String
+        let initiatedDate: Date
+    }
+    let bucket: String
+    let uploads: [S3MultipartUpload]
+
+    static func from(response: AWSResponse) throws -> S3ListMultipartUploadResponse {
+        let doc = try XMLDocument(data: response.data)
```

Comment:
I figured it was simpler to use a DOM parser, especially given the relatively small size of the XML documents returned by the API.

💭  In a future PR, I may consider converting the other `tinys3/responses/*Response.swift` existing files from using the SAX `XMLParser` + `delegate` to this DOM `XMLDocument` parsing approach to simplify the code and have consistency.

Reply:
I'm fine with this, though I wonder if we should put this under test with [an example response](https://github.com/Automattic/hostmgr/blob/trunk/Tests/tinys3Tests/resources/CreateMultipartUpload.xml)?

Reply:
Good point. Will take note to add more tests around this, if not in this PR, then in the follow-up PR where I'll convert existing SAX implementations to DOM.

Reply:
Tests discussed above have been added in https://github.com/Automattic/hostmgr/pull/90